CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing Topics to the Show
+
+## Submitting a Conference
+Submit Upcoming Conferences to be talked about on the show using one our [conference issue templates](https://github.com/python-community-news/topics/issues/new/choose).
```

Comment:
We can link directly to the conference template unless you think we need more than one.

```suggestion
Submit Upcoming Conferences to be talked about on the show using our [conference issue templates](https://github.com/Python-Community-News/Topics/issues/new?assignees=&labels=conference&template=name%3Aadd_conference.yml&title=%5BCONFERENCE%5D+%3CCONFERENCE+NAME%3E+YYYY).
```